installers/msi-language/Uninstall/CustomAction.cs

```diff
@@ -100,7 +106,7 @@ public static ActionResult Dir(Session session, string dir)
                 {
                     Directory.Delete(dir, true);
                 }
-                catch (IOException e)
+                catch (Exception e)
```

Comment:
I think the old one was better, any other exception we'd want to bubble up as we're not handling them..

https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.delete?view=netcore-3.1

Reply:
This stemmed from my testing. When the `perl.exe` binary was being used and I tried to uninstall I got an `UnauthorizedAccessException`... I would expect an `IOException` in this case since that includes `being used by another process` but... Windows 🤷‍♂️

Reply:
huh ok, fair enough then